### DISCUSSION TITLE
Is it possible to install a Lighthouse plugin locally for use within Chrome devtools?

### DISCUSSION THREAD

**Discussion by hursey013** [GoogleChrome/lighthouse#15131] [Q&A]
The Lighthouse tab in Chrome devtools has an option for `Plugins` with `Publisher Ads` as the only option. Is it possible to add my custom Lighthouse plugin to Chrome similar to `Publisher Ads`?

**Comment by connorjclark** [comment]
Currently no, but it is something we've considered doing (via the Chrome Extension store). Can you tell us a little bit about your plugin?

**Comment by hursey013** [comment]
@connorjclark Thanks for your reply! The Lighthouse plugin is for validating certain required elements are present on a webpage. This helps folks to validate they are adhering to whatever web polices are in place when developing a site within that organization. For example, making sure  there is a privacy policy link in the footer, or that a certain branding elements are present, stuff like that. Right now I am experimenting with using the plugin as part of a CI process, but it would be really great to give folks the opportunity to run the audit themselves right in the browser.
